src/keys/rsa.js

```diff
@@ -85,12 +66,12 @@ class RsaPrivateKey {
     return new RsaPublicKey(this._publicKey)
   }
 
-  decrypt (bytes) {
-    return this._privateKey.decrypt(bytes, 'RSAES-PKCS1-V1_5')
+  decrypt (msg, callback) {
+    crypto.decrypt(this._key, msg, callback)
```

Comment:
what is the difference between bytes and msg? Probably node, why change the name?